src/app/feedback/feedback.component.html

```diff
@@ -0,0 +1,45 @@
+<h1 i18n>Add Feedback</h1>
+<form (ngSubmit)="submitfeedback()">
+  <div class="row form-group">
+    <label class="col-4" i18n>Is Your feedback Urgent?</label><br>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4" i18n>
+        <input class="form-check-input" type="radio" value="true" name="isUrgent" [(ngModel)]="this.feedback.isUrgent"> Yes
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4" i18n>
+        <input class="form-check-input" type="radio" value="false" name="isUrgent" [(ngModel)]="this.feedback.isUrgent"> No
+      </label>
+    </div>
+  </div>
+  <div class="row form-group">
+    <label class="col-3" i18n>Type of Feedback</label><br>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-4" i18n>
+        <input class="form-check-input" type="radio" value="Suggestion" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Suggestion
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-3" i18n>
+        <input class="form-check-input" type="radio" value="Bug" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Bug
+      </label>
+    </div>
+    <div class="form-check form-check-inline">
+      <label class="form-check-label col-3" i18n>
+        <input class="form-check-input" type="radio" value="Question" name="feedbackType" [(ngModel)]="feedback.feedbackType"> Question
+      </label>
+    </div>
+  </div>
+  <div class="form-group">
+    <h5 i18n>Feedback</h5>
+    <textarea type="text" name="feedbackMsg"[(ngModel)]="feedback.feedbackMsg" [rows]="7" [cols]="50" required="true"></textarea>
+
+    <div *ngIf="message" [ngClass]="(messageType.pass===true)?'alert alert-success':'alert-danger'" i18n>
+      <span i18n >{message, select, Success { Your feedback is sucessfully submitted}
```

Comment:
shall we have keyword as feedbackSubmitSuccess or something, as success might come up in other components as well.

Reply:
@lmmrssa  what do you mean by 'Need to check which text will be in i18n 'Success' or 'Your feedback....'"?

Reply:
@amanuely regarding that we use i18n to generate xlif file for crowding. So need to see which string will be used in xlif. If its going to be 'Success' then we will have to make it more specific. If its Message itself then it will not be issue.